utils/unittest/fixtures.go

```diff
@@ -2453,3 +2454,35 @@ func ChunkExecutionDataFixture(t *testing.T, minSize int, opts ...func(*executio
 		size *= 2
 	}
 }
+
+func ValidCreateBody(tx flow.TransactionBody) map[string]interface{} {
```

Comment:
I would rename it to smth like `CreateSendTxHttpPayload` otherwise it's not clear what this function does.